.github/workflows/ci-windows.yml

```diff
@@ -204,11 +200,10 @@ jobs:
         cmake --build ${{ github.workspace }}/build --config Release --target mysql_tests
     - name: Test
       run: |
-        $env:NANODBC_TEST_CONNSTR_MYSQL="Driver={MySQL ODBC 8.3 ANSI Driver};Server=127.0.0.1;Database=nanodbc;User=root;Password=;big_packets=1;NO_SCHEMA=false"
+        $env:NANODBC_TEST_CONNSTR_MYSQL="Driver={MySQL ODBC 8.0 ANSI Driver};Server=127.0.0.1;Database=nanodbc;User=root;Password=;big_packets=1;NO_SCHEMA=false"
```

Comment:
This test is failing b/c the tests are not finding the driver. I'm attempting to fix this by adjusting the connection string.